### PR TITLE
Crass 1.0

### PR DESCRIPTION
**Pull request by jmeppley**
:information_source:
Bioconda has finished the [GCC7 migration](https://github.com/bioconda/bioconda-recipes/issues/13578). If you are dealing with C/C++ or Python package it can be that you need to rebuild other dependent packages. [Bioconda utils - update-pinning](https://bioconda.github.io/updating.html#updating-recipes-for-a-pinning-change) will assist you with that. If you have any questions please use [issue 13578](https://github.com/bioconda/bioconda-recipes/issues/13578).

----------------

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by jmeppley** [comment]
Hey, @bioconda/core , what am I missing here? All my tests passed at circleci, but there seems to be a disconnect. The results aren't showing up here. Did I miss a step?

**Comment by dpryan79** [comment]
Disconnects like that happen on occasion. I've retriggered the CI testing.

**Comment by epruesse** [comment]
@biocondabot recheck

**Comment by epruesse** [comment]
@BiocondaBot recheck

**Comment by epruesse** [comment]
@BiocondaBot recheck

**Comment by epruesse** [comment]
@BiocondaBot rebuild

**Comment by epruesse** [comment]
@BiocondaBot rebuild

**Comment by epruesse** [comment]
@BiocondaBot rebuild

**Comment by epruesse** [comment]
@biocondabot rebuild

**Comment by epruesse** [comment]
@BiocondaBot rebuild

**Comment by epruesse** [comment]
@biocondabot rebuild

**Comment by epruesse** [comment]
@BiocondaBot rebuild

**Comment by epruesse** [comment]
@jmeppley Apologies for hijacking your PR to test the bot. I had been waiting for this to happen somewhere so that I could test it. If it occurs again, you should be able to write `@biocondabot rebuild` (at the beginning of a line) to trigger a (re)build on CircleCI.

File: recipes/crass/build.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+./autogen.sh
+./configure --prefix=${PREFIX} --with-xerces=${PREFIX} CPPFLAGS="$CPPFLAGS" LDFLAGS="$LDFLAGS"
```
Comment (epruesse) [inline recipes/crass/build.sh]: Do you need to specify  `CPPFLAGS="$CPPFLAGS"` on the `configure` command line? It should pick those up from the environment directly...

**Comment by epruesse** [comment]
W.r.t. the linker failure - it looks to me like the order is not correct - `-lz` should come after `somelib.a`. This might "happen" to work when `--as-needed` and friends aren't set, but it incorrect and should be repaired upstream or patched in the `configure.ac`, `Makefile.am` & friends files.

More pragmatically ... you could try just adding `export LDFLAGS="$LDFLAGS -lz"` to the `build.sh`. That should append another `-lz` at the end of the linker line and resolve the symbols.

File: recipes/crass/build.sh (hunk 1 above)
Comment (dpryan79) [reply recipes/crass/build.sh]: It doesn't seem to pick up environment variables (truth be told most packages don't seem to), as evidenced by some earlier errors.

**Comment by dpryan79** [comment]
The current build error was fixed in late 2016, but there hasn't been a release since then.

**Comment by dpryan79** [comment]
I don't know why there's a disconnect between the testing on this fork, but the patch I just added should fix the build otherwise. If that's the case I'll just forcibly merge the PR.

**Comment by dpryan79** [comment]
@jmeppley The build now passes, though github isn't aware of that. If you're otherwise happy with the PR I'll merge it.

**Comment by jmeppley** [comment]
That'd be great, thanks!

File: recipes/crass/build.sh (hunk 1 above)
Comment (epruesse) [reply recipes/crass/build.sh]: That's curious. It shouldn't matter to `autotools` built things whether the variable comes from the env or the cmdline.

**Comment by epruesse** [comment]
Let me give it another rebuild.

@biocondabot rebuild

**Comment by epruesse** [comment]
Allright. Thanks everyone!

**Comment by dpryan79** [comment]
Thanks @epruesse !

**Comment by jmeppley** [comment]
Thanks for your help @epruesse & @dpryan79 !

**Pull request merged**